java/README.md

```diff
@@ -0,0 +1,26 @@
+#Word Count in Java
+===================
+Get Vagrant up and running from parent dir:
+	
+	$ ./bootstrap_vagrant.sh
+  	$ vagrant ssh
+  	...
+  	vagrant@vagrant-ubuntu-trusty-64:~/$ cd /vagrant/java/
+  
+Manual SBT install inside Vagrant:
+
+  	$ echo "deb https://dl.bintray.com/sbt/debian /" | sudo tee -a /etc/apt/sources.list.d/sbt.list
```

Comment:
I spent a little time trying to add SBT via ansible.  I will dig a little more on that.